app.js

```diff
@@ -1,8 +1,12 @@
 const express = require("express");
 const app = express();
-const { getTopics } = require("./controllers/topics.controller")
-const { handleCustomErrors, handle404Errors, handle500Errors } = require("./error-handling")
+const { getTopics } = require("./controllers/topics.controller");
+const { handleCustomErrors, handle404Errors, handle500Errors } = require("./error-handling");
+const endpoints = require("./endpoints.json");
 
+app.get("/api", (request, response) => {
+    response.status(200).send({ endpoints: endpoints})
```

Comment:
Data shouldn't be imported into the app, so we need a controller function here 👀 
If the data is imported into the app, it exists on the memory of the server all the time, even if a request isn't made to the endpoint that responds with it. This is inefficient for the performance of our server.